### PR TITLE
dnsChallenge LAN changes

### PR DESCRIPTION
**Pull request by leakypixel**
While using this addon to make a dnsChallenge certificate request, I ran into a couple of issues relating to DNS on my LAN not mirroring that of the public internet, as well as special characters in API keys. 

In order to make this work for me, I had to make a couple of changes to allow escape characters in env vars, and add delayBeforeCheck and resolvers to the generated traefik static config.

I also tidied up a little around the previous PR I made, by adding the references to aarch64 in the config and build files.

### CODE REVIEW

File: traefik/traefik.yaml.j2 [hunk 1]
```diff
@@ -33,6 +33,14 @@ certificatesResolvers:
   {%- elif letsencrypt.challenge_type == 'dnsChallenge' %}
       dnsChallenge:
         provider: {{ letsencrypt.provider }}
+        delayBeforeCheck: {{ letsencrypt.delayBeforeCheck }}
+      {% if letsencrypt.resolvers %}
+        resolvers:
+        {% for resolver in letsencrypt.resolvers -%}
+          - "{{ resolver }}"
+
+        {% endfor %}
```
Comment (alex3305) [inline traefik/traefik.yaml.j2:42]: Please remove the empty line and you can add an `-` to the `endfor` within the yaml loop to keep it all neatly tight together.

```suggestion
        {%- endfor %}
```

**Review by alex3305 — CHANGES_REQUESTED** [review]
Great changes! Thanks for that. I have some small review comments I would like to iron out before merging.

I would also like to see some documentation on the new parameters within the [Readme](README.md) file.

File: traefik/config.json [hunk 2]
```diff
@@ -3,7 +3,11 @@
     "version": "3.0.3",
     "slug": "traefik",
     "description": "Traefik is a modern HTTP reverse proxy and load balancer that makes deploying microservices easy.",
-    "arch": ["armv7", "amd64"],
+    "arch": [
```
[the remark this answers is not in the document]
Comment (leakypixel) [reply traefik/config.json:6]: No problem - to be honest I didn't make this change deliberately, I'll make these changes now :)

**Comment by leakypixel** [comment]
I believe that's the changes done - apologies for the reformatting, it was a complete oversight. 

Thanks for taking the time to review this, let me know if there're any other changes required :)

**Comment by alex3305** [comment]
Thanks for the changes you have made. I have accepted your PR.

When I requested additional documentation I thought that the docs were still in the [Readme](README.md), but I forgot that they were moved to the [Docs](DOCS.md) file. I will move them for you. I will also update the add-on version, so your changes will be available through an update of this add-on.

Thanks again for your contribution and the nice collaboration!

**Pull request merged**